### DISCUSSION TITLE
Website redesign

### DISCUSSION THREAD

**Discussion by JYamihud** [tenacityteam/tenacity-legacy#162] [Ideas]
# Current website
![2021-07-08_17-18](https://user-images.githubusercontent.com/27649397/124938136-b83c7300-e010-11eb-82bb-bd06858d8c17.png)
This is a very basic text site with a few problems. 

 - Words like Repository, Mailing Lists and IRC are understood only by very advanced users. I'd change them to `Download` and `Community`. 
 - The website is not very attractive for potential users.
 - There is only one page on the website. More things should be done from the site directly. 
# Design 2.0

![tenacity-site](https://user-images.githubusercontent.com/27649397/124938603-1a957380-e011-11eb-9295-67188eb05036.png)

I made this picture in GIMP in a few minutes. Which already reads like a way better looking website. After features, icons that describe features, with a quick explanation of them, could be added. 

We could copy the aesthetic of the www.audacityteam.org site. Or could make it look more like the list. Even a special page for Features could be added. Focusing on latest news on the main page. But then somebody will need to write those news articles. 

A pretty screenshot of Tenacity is required. I used Audacity 2.3 for this image. But a true Tenacity screenshot with a true tenacity theme should be the one shown.

**Comment by fossdd** [comment]
The website is probibly not the final one. Your image looks fancy, but I think we should keep it to the information. And there are not multiple pages required. Different Fonts is imo too much, we should the browser decide which font he want.

The source code to the page is open, see https://git.sr.ht/~tenacity/tenacityaudio.org. Everyone can send patches in.

BTW: Currently Tenacity doesn't look different then Audacity. In the moment the changes are only that telemetry, crash reports and other shenanigans are removed.

**Comment by falkTX** [comment]
Let's discuss this in the ticket, since tickets can be archived but discussions cannot.
Thanks.
See #163

**Reply by Semisol** [reply]
Fixed a spelling mistake ;)
